contracts/cluster_agent/src/entry.rs

```diff
@@ -111,7 +111,7 @@ fn process_transfer() -> Result<(), Error> {
     let input_agent_data = load_cell_data(0, GroupInput)?;
     let output_agent_data = load_cell_data(0, GroupOutput)?;
 
-    if input_agent_data != output_agent_data {
+    if input_agent_data != output_agent_data || input_agent_data.is_empty() {
```

Comment:
https://github.com/sporeprotocol/spore-contract/blob/feat/complete-test-cases/contracts/cluster_agent/src/entry.rs#L47-L50

input_agent_data should be proxy_type_hash, a.k.a Hash(ClusterProxyCell.Type).
So, it won't be empty.

Reply:
yes, completely, I treat it as a double check

Reply:
LGTM